scripts/weekly-tileset-generation.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+
+import hyp3_sdk as sdk
+import requests
+from get_frame_list_from_cmr import get_frames_for_direction
+
+
+def publish_mosaic(job: sdk.Job) -> None:
+    source = f's3://hyp3-opera-disp-sandbox-contentbucket-ibxz8lcpdo59/{job.job_id}/tms/'
+    target = f's3://asf-services-web-content-prod/main/{job.name}/'
+    subprocess.run(['aws', '--profile', 'edc-prod', 's3', 'sync', source, target], check=True)
```

Comment:
Trial run suggests using `sync` doesn't buy us anything here. Since .pngs for the new mosaic will all more recent file modification dates, `sync` will copy all of them, even if the content of those .pngs hasn't changed:

> An S3 object will require copying if the sizes of the two S3 objects differ, the last modified time of the source is newer than the last modified time of the destination, or the S3 object does not exist under the specified bucket and prefix destination.

I think switching to `cp --recursive` would be slightly more performant.

Reply:
@asjohnston-asf are you leaning one way over the other?

Reply:
I added the delete tag for the time being but can easily switch to cp if that's how we'd prefer.